operations/vouchers.md

```diff
@@ -210,63 +237,62 @@ Adds the specified vouchers to the specified [Services](services.md#service). No
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
-| `VoucherParameters` | array of [Voucher parameters](#voucher-parameters) | required, max 1000 items | Vouchers to be added. |
+| `EnterpriseId` | string | optional | Unique identifier of the enterprise. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
+| `VoucherParameters` | array of [Voucher Parameters](vouchers.md#voucher-parameters) | required, max 1000 items | Vouchers to be added. |
 
-#### Voucher parameters
+#### Voucher Parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Id` | string | required | Unique identifier of voucher. |
-| `ServiceId` | string | required | Unique identifier of [Service](services.md#service) the voucher belongs to. |
-| `Name` | string | required, max 128 characters | Internal name of the voucher. |
-| `Type` | string [Voucher type](#voucher-type)| required | Type of the voucher. |
-| `CompanyId` | string | optional | Unique identifier of [Company](companies.md#company) \(Company or Travel Agency\) the voucher is related to. This is _required_ for [Type](#voucher-type) of `PartnerCompany` or `TravelAgency`. |
-| `AssignedRateIds` | array of string | optional, max 5 items | Unique identifiers of [Rates](rates.md#rate). |
-| `OccupiableIntervalStartUtc` | string | required | Start of the interval in which the voucher can be applied. |
-| `OccupiableIntervalEndUtc` | string | required | End of the interval in which the voucher can be applied. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the voucher from external system. |
+| `ServiceId` | string | required | Unique identifier of Service. |
+| `Name` | string | required, max length 128 characters | Internal name of the voucher. |
+| `Type` | [Voucher Type](vouchers.md#voucher-type) | required | Type of the voucher. |
+| `CompanyId` | string | optional | Unique identifier of Company. |
+| `AssignedRateIds` | array of string | optional, max 5 items | Unique identifiers of Rates. |
+| `OccupiableIntervalStartUtc` | string | optional | Start of the interval in which the voucher can be applied. |
+| `OccupiableIntervalEndUtc` | string | optional | End of the interval in which the voucher can be applied. |
```

Comment:
❓: `OccupiableIntervalStartUtc` and `OccupiableIntervalEndUtc` were originally `required` but from code it seems like it supposed to be `optional`. Let me know if you think if it should be `required`.

Reply:
You're correct, the value is marked as nullable in the entity and should be marked as nullable in DTO as well.